provider/manual/environ.go

```diff
@@ -239,26 +239,36 @@ func (e *manualEnviron) DestroyController(controllerUUID string) error {
 	script := `
 set -x
 touch %s
```

Comment:
I think it would be handy to use numbered args as it is easier to correlate with the Sprintf arguments. e..g., use %[1]s or %3[q], etc.